bundles/ch.elexis.core/src/ch/elexis/core/model/PatientConstants.java

```diff
@@ -6,8 +6,6 @@
 	public static final String FLD_EXTINFO_DATE_OF_DEATH = "DateOfDeath";
 	public static final String FLD_EXTINFO_PROFESSION = "Profession";
 	public static final String FLD_EXTINFO_BIRTHNAME = "BirthName";
-	public static final String FLD_EXTINFO_INCREASEDTREATMENT = "IncreasedTreatment";
```

Comment:
Weshalb wird das hier entfernt?

Reply:
falsche Änderung, sollte nicht in diesem Pull request sein